jetty-core/jetty-http2/jetty-http2-common/src/main/java/org/eclipse/jetty/http2/HTTP2Stream.java

```diff
@@ -190,6 +193,16 @@ public void reset(ResetFrame frame, Callback callback)
         }
     }
 
+    /**
+     * Ask to be called back when all the frames of this session have been
+     * flushed to the network.
+     * @param callback the callback that gets notified when the frames have been flushed
+     */
+    public void flush(Callback callback)
```

Comment:
Remove this method, as it is not a "stream" flush, but a "session" flush.
The caller can just do: `stream.getSession().flush(cb)`.